src/Castle.Facilities.Synchronize/Castle.Facilities.Synchronize-VS2017.csproj

```diff
@@ -33,4 +33,8 @@
   <ItemGroup>
     <Compile Include="..\..\buildscripts\CommonAssemblyInfo.cs" />
   </ItemGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)'=='net45'">
+    <DefineConstants>$(DefineConstants);FEATURE_WINFORMS</DefineConstants>
```

Comment:
I'm curious why you defined `FEATURE_WINFORMS` in here, we're very unlikely to ever want to build the .NET Framework builds without this code and it is surfaced more clearly if they are all in `common.props`.